### DISCUSSION TITLE
Can you add IAP on Google Play Store without Rolling Out to Production?

### DISCUSSION THREAD

**Discussion by gabeDevlift** [hyochan/react-native-iap#1360] [Q&A]
I've been following the docs on how to add the IAP items for Android, and it seems like you have to click "Rollout to Production" on the Google Play Console before you can actually add the items. I want to be able to test all my code before pushing the app to the store; is there a way to add the items to the Console before Rolling Out?

**Comment by hyochan** [comment]
My last understanding of this is that you do not have to `Rollout to production` for the development. You only need to upload your `apk` or `appbundle` and this can be done privately (ex. `alpha channel`).
